### PR TITLE
changed css styles

### LINKED ISSUE #475: CSS styles for the new version (GSoD2020) of the Interactive Book
## Describe the bug

The new version of the book (being developed under the GSoD 2020 project) replaced the old version Just-the-docs (JTD) Jekyll theme, which was installed locally and included styling and structure hardcoded into it, with the current upstream JTD version installed as a Gem/plugin for Jekyll. In the process some of the original style of the book was lost, either because it was hardcoded in the template, or because it was closely related to document structures which are not now part of theme.

At least, the following problems with the style have been identified (there might be more). Check the attached screenshot for details (left is the pre-GSoD2020 version, right is the new version):

1. In the navigation bar, single pages at the top level and chapters (group of pages) have different "indentation" ("yellowish" 1 in the screenshot).
2. In the navbar, the distance from the bullet marker to the text is larger than in the original design (red 2 in the screenshot).
3. Chapter title is located more to the right than the sections in the chapter. Also, the section title style seems be a bit of with respect to the rest of the nav bar: i.e. color, marker, etc., although it is the same in the current version's style. (blue 3 in the screenshot).
4. Links at the top right are no longer buttons (they are part of the JTD theme) but they can be styled as such, it just need the proper CSS code (red 4 in the screenshot).
5. Table of content headline doesn't match the original book style. (Blue 5 in the screenshot).

## Expected behavior

Consistent style and closer to the current one

## Screenshots

CVIB_versus-2020-10-07_labels

## Additional context

All css should go in the `_sass/custom/custom.scss` as indicated in the JTD [documentation](https://pmarsceill.github.io/just-the-docs/docs/customization/#override-and-completely-custom-styles)

See PR #474

### PR DESCRIPTION
**Pull request by anushri20**
CSS Styles for the new version GSoD2020

Fixes #475 

#### Changes done:
- [x] Changes in custom.scss and markdown files

#### Screenshots
Old 
image

New
image

#### ✅️ By submitting this PR, I have verified the following

- [ x] Checked to see if a similar PR has already been opened 🤔️
- [ x] Reviewed the contributing guidelines 🔍️
- [x ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by manjotsidhu** [comment]
@anushri20 are you active to work further on this PR ?

**Comment by manjotsidhu** [comment]
We have to fix more stuff in this PR, therefore I would be extending this PR to incorporate the enhancements.

**Comment by anushri20** [comment]
@manjotsidhu  Hi, yes i am active to work further on this PR.

**Comment by manjotsidhu** [comment]
> @manjotsidhu Hi, yes i am active to work further on this PR.

Great, first you can rebase your changes as there are a lot of updates on that branch.

File: Gemfile.lock [hunk 1]
```diff
@@ -26,10 +26,12 @@ GEM
     ethon (0.12.0)
```
Comment (manjotsidhu) [inline Gemfile.lock:26]: Please exclude changes done in this file

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by manjotsidhu** [comment]
Add `.text-delta` class to `utils/create_project.py`, line 44 and line 53.

**Comment by anushri20** [comment]
Okay.

**Comment by Shivansh2407** [comment]
@manjotsidhu @anushri20 you can work and get this PR done soon, in order to complete the CSS fixes for the GSOD 2020. Let me know if you have any issues/doubts! Happy Contributing 🎉

**Comment by manjotsidhu** [comment]
@anushri20 Can we get updates on this ?

**Comment by anushri20** [comment]
@manjotsidhu Sure, I will make all the changes , but I am busy right now due to college. So can I do it after two days?

**Comment by manjotsidhu** [comment]
> @manjotsidhu Sure, I will make all the changes , but I am busy right now due to college. So can I do it after two days?

Actually there a lot of CSS changes we need to complete. I can make a seperate Pull Request and proceed further on top of your changes. Is it fine on your end ?

**Comment by anushri20** [comment]
> 
> 
> > @manjotsidhu Sure, I will make all the changes , but I am busy right now due to college. So can I do it after two days?
> 
> Actually there a lot of CSS changes we need to complete. I can make a seperate Pull Request and proceed further on top of your changes (including authorship). Is it fine on your end ?

Okay, let me see. These are the only changes that you need?

**Comment by anushri20** [comment]
@manjotsidhu I have made the above changes, please let me know if there are any other changes. Thank you

**Comment by manjotsidhu** [comment]
@anushri20 there are merge conflicts since this PR is old. You need to fix these conflicts, exclude `Gemfile.lock`, and fix hound issues.

**Comment by anushri20** [comment]
> 
> 
> @anushri20 there are merge conflicts since this PR is old. You need to fix these conflicts, exclude `Gemfile.lock`, and fix hound issues.

i can't find any conflicts or issue. Is it fine if i just create a new PR?

**Comment by manjotsidhu** [comment]
@Shivansh2407 Should be closed since #516 merged.

**Comment by manjotsidhu** [comment]
Can be closed, superseded by #528

**Comment by manjotsidhu** [comment]
> Can be closed, superseded by #528

@Shivansh2407

**Comment by Shivansh2407** [comment]
> 
> 
> Can be closed, superseded by #528

**Pull request closed without merging**